client/Brocfile.js

```diff
@@ -1,11 +1,23 @@
-/* global require, module */
-
+/* global require, module, process */
+var env = process.env.EMBER_ENV;
```

Comment:
Is there a way to get the ember environment some other way than by an environment variable? That seems sub-optimal.